src/main/java/de/tum/in/www1/artemis/web/rest/AccountResource.java

```diff
@@ -101,7 +101,7 @@ public String isAuthenticated(HttpServletRequest request) {
     @GetMapping("/account")
     public UserDTO getAccount() {
         log.info("GET /account " + SecurityUtils.getCurrentUserLogin().get());
-        return Optional.ofNullable(userService.getUserWithAuthorities()).map(UserDTO::new).orElseThrow(() -> new InternalServerErrorException("User could not be found"));
+        return Optional.ofNullable(userService.getUserWithGroupsAndAuthorities()).map(UserDTO::new).orElseThrow(() -> new InternalServerErrorException("User could not be found"));
```

Comment:
The UserService getUserWithAuthorities() was called which was a duplicate method to getUserWithGroupsAndAuthorities(). I changed the call here and removed getUserWithAuthorities() in the UserService class.